Cargo.toml

```diff
@@ -2,32 +2,32 @@
 panic = "unwind"
 
 [patch."https://github.com/sora-xor/sora2-common.git"]
-# beefy-light-client = { path = "../sora2-common/pallets/beefy-light-client" }
-# beefy-light-client-rpc = { path = "../sora2-common/pallets/beefy-light-client/rpc" }
-# beefy-light-client-runtime-api = { path = "../sora2-common/pallets/beefy-light-client/runtime-api" }
-# leaf-provider = { path = "../sora2-common/pallets/leaf-provider" }
-# leaf-provider-rpc = { path = "../sora2-common/pallets/leaf-provider/rpc" }
-# leaf-provider-runtime-api = { path = "../sora2-common/pallets/leaf-provider/runtime-api" }
-# bridge-common = { path = "../sora2-common/pallets/bridge-common" }
-# dispatch = { path = "../sora2-common/pallets/dispatch" }
-# bridge-types = { path = "../sora2-common/pallets/types" }
-# bridge-data-signer = { path = "../sora2-common/pallets/data-signer" }
-# multisig-verifier = { path = "../sora2-common/pallets/multisig-verifier" }
-# substrate-bridge-channel = { path = "../sora2-common/pallets/substrate-channel" }
-# bridge-channel-rpc = { path = "../sora2-common/pallets/substrate-channel/rpc" }
-beefy-light-client = { git = "https://github.com/sora-xor//sora2-common.git", tag = "1.0.8" }
-beefy-light-client-rpc = { git = "https://github.com/sora-xor//sora2-common.git", tag = "1.0.8" }
-beefy-light-client-runtime-api = { git = "https://github.com/sora-xor//sora2-common.git", tag = "1.0.8" }
-leaf-provider = { git = "https://github.com/sora-xor//sora2-common.git", tag = "1.0.8" }
-leaf-provider-rpc = { git = "https://github.com/sora-xor//sora2-common.git", tag = "1.0.8" }
-leaf-provider-runtime-api = { git = "https://github.com/sora-xor//sora2-common.git", tag = "1.0.8" }
-bridge-common = { git = "https://github.com/sora-xor//sora2-common.git", tag = "1.0.8" }
-dispatch = { git = "https://github.com/sora-xor//sora2-common.git", tag = "1.0.8" }
-bridge-types = { git = "https://github.com/sora-xor//sora2-common.git", tag = "1.0.8" }
-bridge-data-signer = { git = "https://github.com/sora-xor//sora2-common.git", tag = "1.0.8" }
-multisig-verifier = { git = "https://github.com/sora-xor//sora2-common.git", tag = "1.0.8" }
-substrate-bridge-channel = { git = "https://github.com/sora-xor//sora2-common.git", tag = "1.0.8" }
-bridge-channel-rpc = { git = "https://github.com/sora-xor//sora2-common.git", tag = "1.0.8" }
+beefy-light-client = { path = "../sora2-common/pallets/beefy-light-client" }
```

Comment:
please, rollback this file

Reply:
> please, rollback this file

shure, I will change it after merging this https://github.com/sora-xor/sora2-common/pull/50 and creating tag